### PR TITLE
Increase code coverage

### PR DESCRIPTION
**Pull request by martijnversluis**
Let's increase the code coverage to 💯%!!

### CODE REVIEW

File: spec/support/utilities.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def capture(stream)
+  begin
+    stream = stream.to_s
+    eval "$#{stream} = StringIO.new"
+    yield
+    result = eval("$#{stream}").string
+  ensure
+    eval("$#{stream} = #{stream.upcase}")
```
Comment (drumusician) [inline spec/support/utilities.rb]: waarom upcase als je case insensitive checkt?

Comment (martijnversluis) [reply spec/support/utilities.rb]: [Hierom](https://github.com/kabisa/launch-base/blob/0bc6d0cec5a79228ed1e1105aa5ebc9b900d541d/spec/support/utilities.rb#L1)

Comment (nickve28) [reply spec/support/utilities.rb]: waarom nederlandse comments op engelse code

Comment (drumusician) [reply spec/support/utilities.rb]: :shipit: 🚀

**Pull request merged**